misc/api-doc.spec.js

```diff
@@ -1,69 +1,68 @@
-var expect = require('chai').expect;
 var apiDoc = require('./api-doc');
 
-describe('API doc extractor', function() {
+describe('APIDocVisitor', function() {
 
   it('should return [] if there are no docs to extract',
-     function() { expect(apiDoc(['./misc/api-doc-test-cases/no-docs.ts'])).to.be.empty; });
+     function() { expect(apiDoc(['./misc/api-doc-test-cases/no-docs.ts'])).toEqual({}); });
 
   it('should extract basic info from directives and components', function() {
     var docs = apiDoc(['./misc/api-doc-test-cases/directives-no-in-out.ts']);
 
-    expect(Object.keys(docs)).to.have.lengthOf(2);
+    expect(Object.keys(docs).length).toEqual(2);
 
-    expect(docs.Foo.fileName).to.equal('misc/api-doc-test-cases/directives-no-in-out.ts');
-    expect(docs.Foo.className).to.equal('Foo');
-    expect(docs.Foo.selector).to.equal('[foo]');
-    expect(docs.Foo.description).to.equal('Foo doc');
-    expect(docs.Foo.exportAs).to.equal('foo');
+    expect(docs.Foo.fileName).toEqual('misc/api-doc-test-cases/directives-no-in-out.ts');
```

Comment:
`toEqual` => `toBe`:
* `toEqual` is the same as `==` in JS
* `toBe` is the same as `===` in JS

The same comment applies to all the remaining checks for primitive values.